src/engine/SCons/Tool/msvc.xml

```diff
@@ -475,4 +475,38 @@ Valid values are '1' or '0'
 </summary>
 </cvar>
 
+<cvar name="VSWHERE">
+<summary>
+<para>
+Specify the location of vswhere.exe. 
+</para>
+
+<para>
+<literal>vswhere.exe</literal> is used to locate MSVC/MSVS installations.
+It was initially distributed with MSVS 2017. But can detect MSVC 2010 and newer with <literal>-legacy</literal> argument. 
+If &cv-VSWHERE; is set, SCons will use that location.
+</para>
```

Comment:
Suggest move the paragraph break up one line - "if set" and "otherwise" are part of the same thought.